requirements.txt

```diff
@@ -19,6 +19,7 @@ decorator==4.3.0
 defusedxml==0.5.0
 dj-database-url==0.5.0
 Django==2.1.1
+django-cached-field==1.4.0
```

Comment:
Do we need this library? Would rather avoid having this dependency and all of the side effects:

```
# (from their list of caveats)
-All ORM methods (e.g. order_by, filter) can only access this field through cached_FIELD.
- recalculate_FIELD uses .update(cached_FIELD= to set the value. Don't expect .save to be called.
- flag_FIELD_as_stale uses .update, as well.
- This may break if you try to add this mixin to a field class that multiply-inherits (I'm currently grabbing an arbitrary, non-CachedFieldMixin class and making the real field with it).
- The FIELD_recalculation_needed field is accessed by regex in at least one place, so problems will result from user fields that match the same pattern.
- Exceptions in a post-hook will potentially break saves of the data.
```

Reply:
Replacing with cached @property on model. @ckcollab